src/util/MapPolyfill.ts

```diff
@@ -20,7 +20,7 @@ export class MapPolyfill {
     return this;
   }
 
-  delete(key): boolean {
+  delete(key: any) {
```

Comment:
Ah yes, seems you're correct. It's been added recently after this PR.. Please just rebase to reflect latest status then. :) Thanks for pointing.